install/install.sh

```diff
@@ -154,12 +157,24 @@ echo ""
 echo "We need some information for the admin account"
 read -e -r -p "Admin username: " -i "admin" admin_name
 read -e -r -p "Admin email: " admin_email
-read -e -r -p -s "Admin password: " admin_password
-read -e -r -p -s "Confirm admin password: " confirm_admin_password
-while [ $admin_password -ne $confirm_admin_password ]; do
+while [$admin_email -z ];do
+   echo "Entered email can't be empty!"
+   echo "Please Re-enter email again ... "
+   read -e -r -p "Admin email: " admin_email
+done 
+read -e -r -p  "Admin password: " admin_password
+read -e -r -p  "Confirm admin password: " confirm_admin_password
+while [$admin_password -z ];do
+   echo "Entered password can't be empty!"
+   echo "Please Re-enter password again ... "
+   read -e -r -p  "Admin password: " admin_password
+   read -e -r -p  "Confirm admin password: " confirm_admin_password
+
+done 
+while [ $admin_password != $confirm_admin_password ]; do
     echo "Entered passwords did not match! Retrying..."
-    read -e -r -p -s "Admin password: " admin_password
-    read -e -r -p -s "Confirm admin password: " confirm_admin_password
+    read -e -r -p "Admin password: " admin_password
```

Comment:
Same question here.

Reply:
it has a much worse effect here as it ends up creating a loop of your passwords do not match at this place and blocks the input